src/commcare_cloud/ansible/roles/kafka/files/log4j_2.6.0.properties

```diff
@@ -0,0 +1,86 @@
+log4j.rootLogger=INFO, stdout, kafkaAppender
```

Comment:
why do we need a new log config file? Are there significant changes?

Reply:
it because :  


- name: Copies kafka log4j.properties
  copy:
    src: "log4j_**{{ kafka_version }}**.properties"
    dest: /opt/kafka/config/log4j.properties
    owner: root

Reply:
maybe we should just change that to be a fixed filename and use a when condition `when: kafka_version is version('2.0', '>')`

Reply:
Done  the changes.